.circleci/config.yml

```
@@ -833,6 +833,66 @@ jobs:
           path: tmp/capybara
           destination: capybara
 
+  check_production_gemfile:
+    parameters:
+      executor:
+        type: string
+        default: builder-ruby25
+    executor:
+      name: << parameters.executor >>
+      database: mysql
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "d8:d6:e4:53:b3:0e:22:76:01:e0:27:90:a6:c3:fa:df"
+      - checkout-with-submodules
+      - restore-gem-cache
+      - run:
+          name: Keep production gemfile up to date
+          command: |
+            echo "keeping production gemfiles up to date"
+            git config user.email "3scale@redhat.com"
+            git config user.name "3scale"
+            unset BUNDLE_FROZEN
+            bundle install --no-deployment
+            make bundle
+            FILES_CHANGED=$(git status --porcelain | sed s/^...//)
+            [ -z "$FILES_CHANGED" ] && exit 0
```

Use double brackets to protect your shell, though here it right to use single one


```suggestion
            [[ -z "$FILES_CHANGED" ]] && exit 0
```